### PR TITLE
Insert matching OGV grant script

### PR DESCRIPTION
**Pull request by tomlinton**

### CODE REVIEW

**Comment by franckc** [comment]
hey @shahthepro , could you help me review this script before I run it in production? thanks...

File: token-transfer-client/src/components/VestingHistory.js [hunk 1]
```diff
@@ -20,15 +20,17 @@ const VestingHistory = props => {
   })
 
   const tableRows = []
+  let key = 0
```
Comment (franckc) [inline token-transfer-client/src/components/VestingHistory.js:23]: We should probably commit this as a separate PR. It's completely unrelated.

File: token-transfer-server/src/scripts/insert_ogv_matching_grants.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+Logger.setLogLevel(process.env.LOG_LEVEL || 'INFO')
+const logger = Logger.create('insert_ogv_airdrop_grants', {
+  showTimestamp: false
+})
+
+const SNAPSHOT_DATE = '2022-7-12'
+
+async function run() {
+  const users = await db.User.findAll({ where: { id: 693 } })
```
Comment (franckc) [inline token-transfer-server/src/scripts/insert_ogv_matching_grants.js]: We need to remove this where clause. Must have been for testing.

File: token-transfer-server/src/scripts/insert_ogv_matching_grants.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+      const grantStartDate = moment.min(...filteredVesting.map(v => v.date))
+      const grantEndDate = moment.max(...filteredVesting.map(v => v.date))
+
+      const newGrant = {
+        userId: user.id,
+        start: grantStartDate,
+        cliff: grantStartDate,
+        end: grantEndDate,
+        amount: totalGrantAmount.toString(),
+        // TODO: Not sure if T3 will correctly handle cliff that is the same as grant start date
```
Comment (franckc) [inline token-transfer-server/src/scripts/insert_ogv_matching_grants.js]: Hmm. Should test that before I run it in prod.

File: token-transfer-server/src/scripts/insert_ogv_matching_grants.js [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+const logger = Logger.create('insert_ogv_airdrop_grants', {
+  showTimestamp: false
+})
+
+const SNAPSHOT_DATE = '2022-7-12'
+
+async function run() {
+  const users = await db.User.findAll({ where: { id: 693 } })
+  for (const user of users) {
+    const grants = await db.Grant.findAll({ where: { userId: user.id } })
```
Comment (shahthepro) [inline token-transfer-server/src/scripts/insert_ogv_matching_grants.js]: nit: Could actually use `Grant` model in `includes` attribute in the `User.findAll` query above instead

File: token-transfer-server/src/scripts/insert_ogv_matching_grants.js (hunk 3 above)
Comment (shahthepro) [reply token-transfer-server/src/scripts/insert_ogv_matching_grants.js]: If you don't mind, I can do this testing today

**Comment by franckc** [comment]
@shahthepro I made a few small changes and also fixed the unit tests. Could you please review before I run this in prod? Thanks!

**Review by franckc — APPROVED** [review]

**Pull request merged**